docs/docker.md

```diff
@@ -39,5 +35,8 @@ Besides, currently `cvdr` supports using SOCKS5 proxy with the flag like
 
 ## Use cloud orchestrator by cvdr
 
-Please follow [cvdr.md](cvdr.md). The URL of running cloud orchestrator should
-be `http://${CLOUD_ORCHESTRATOR_IP_ADDRESS}:8080`.
+Please use the configuration file `scripts/docker/cvdr.toml` and follow the
+steps at [cvdr.md](cvdr.md). For example:
+```bash
+CVDR_USER_CONFIG_PATH=./scripts/docker/cvdr.toml ./cvdr host create
```

Comment:
Sorry for late reply, could we mention how to use `cvdr` into `docs/cvdr.md`, not `docs/docker.md`? Also, could you remove all `--service_url`s in `docs/cvdr.md` and replace with `CVDR_USER_CONFIG_PATH`?

The main reason for this suggestion is `docs/cvdr.md` suggests to find `SERVICE_URL` value, but nowhere(docs/cvdr.md`, `docs/cloud_orchestrator.md`, `docs/docker.md`) doesn't explain what's `SERVICE_URL`.

Reply:
I think we would still need a `cvdr` section at `docs/docker.md`. This is because we have a specific `./scripts/docker/cvdr.toml` for the docker use case.

And regarding replacing `--service_url`s in `docs/cvdr.md` with `CVDR_USER_CONFIG_PATH`, which `cvdr.toml` would you suggest us to use? Should we declare a new one or should we just refer to `build/package/host/etc/cvdr.toml`?

Reply:
> I think we would still need a cvdr section at docs/docker.md.

Agree, I thought there still needs `cvdr` section, but thought no needs to put extra information beyond explaining the path of `CVDR_USER_CONFIG_PATH`. Then, `docs/cvdr.md` should take the responsibility explaining about how to use `CVDR_USER_CONFIG_PATH`.

> And regarding replacing --service_urls in docs/cvdr.md with CVDR_USER_CONFIG_PATH, which cvdr.toml would you suggest us to use?

As I wrote about SERVICE_URL before, I think `docs/cvdr.md` needs a section providing the redirection link towards like `docs/docker.md`. For instance, setting the title of section as `Prepare cvdr.toml`, and giving a suggestion with that link.

Reply:
SGTM! Done.

Reply:
Thanks :)